nixos/modules/services/monitoring/scrutiny.nix

```diff
@@ -36,6 +37,8 @@ in
           Scrutiny settings to be rendered into the configuration file.
 
           See https://github.com/AnalogJ/scrutiny/blob/master/example.scrutiny.yaml.
+
+          Options containing secret data should be set to an attribute set containing the attribute `_secret` - a string pointing to a file containing the value the option should be set to.
```

Comment:
> ... a string ...

Since https://github.com/NixOS/nixpkgs/pull/325217, genJqSecretsReplacementSnippet also supports structured (json) secrets.  Not sure if this info is useful here though.